src/commands/fun/meme.js

```diff
@@ -117,30 +117,29 @@ async function getRandomEmbed(choice) {
   const subReddits = ["meme", "Memes_Of_The_Dank", "memes", "dankmemes"];
   let rand = choice ? choice : subReddits[getRandomInt(subReddits.length)];
```

Comment:
change this too
```javascript
let rand = choice ? choice : subReddits[getRandomInt(subReddits.length - 1)]
``` 
Array length is starts from 1 but iterating an array starts from `0`,  so if the randomInt is by any chance equal to array length, it'll return undefined as no. 4 would be for 5th element in the array which doesn't exist here.

This issue was first pointed by Rhaym